web/public/css/profile-image.less

```diff
@@ -0,0 +1,23 @@
+@media (max-width:767px){
+	.mobile-center {
+		margin: auto;
+		width: 50%;
```

Comment:
I get the thought process, but the cd-round-image has a width of 200px, which can overflow this 50% when at resolutions like 320px (iPhone 4/S, 5/S, SE all have this width). 200px is a nice size on mobile anyway, so I'd recommend just removing this width, and having the 200px width stay as is.